src/molecules/Tabs.tsx

```diff
@@ -92,25 +82,25 @@ const Tabs: React.FC<TabsPros> = ({
           path={match.url}
           exact
           render={() => (
-            <StyledP>
+            <Grid>
```

Comment:
If all your routes are wrapped in `Grid`, it's the same as wrapping the while `<Switch>` in `Grid`:

```jsx
<Grid>
  <Switch>
    <Route {...} />
    <Route {...} />
  </Switch>
</Grid>
```